include/openssl/ssl.h

```diff
@@ -1463,8 +1463,20 @@ __owur BIO *BIO_new_buffer_ssl_connect(SSL_CTX *ctx);
 __owur int BIO_ssl_copy_session_id(BIO *to, BIO *from);
 void BIO_ssl_shutdown(BIO *ssl_bio);
 
-__owur int SSL_CTX_set_cipher_list(SSL_CTX *, const char *str);
 __owur SSL_CTX *SSL_CTX_new(const SSL_METHOD *meth);
+/* As an option for apps, TLSv1.3 can be disabled */
+#ifndef OPENSSL_DISABLE_TLSv13
```

Comment:
Is that a lower case v? Seems not quite right when everything else is in caps.

Reply:
You originally suggested `DISABLE_TLSv13`, with lower case v.  However, considering all other macros, this should really be `OPENSSL_DISABLE_TLS1_3`, should it not?

Reply:
Oh right...well that was just an example :-)

Yes, dropping the "v" seems best.